prometheus_juju_exporter/config.py

```diff
@@ -20,16 +22,16 @@ def __init__(self, args=None):
         self.validate_config_options()
 
     @property
-    def config(self):
+    def config(self) -> confuse.Configuration:
         """Return the configuration parsed from the config file."""
-        global config
+        global CONFIG  # pylint: disable=W0603
```

Comment:
Do you want to implement singleton pattern?
https://refactoring.guru/design-patterns/singleton/python/example

Reply:
I agree that singleton patter is better here, but it's out of scope for this PR.

Reply:
I have adopted the singleton design pattern for the configuration class. This is the first time I'm using it so hopefully I'm doing it in a correct way (all tests passed).